src/lib.rs

```diff
@@ -312,6 +319,12 @@ impl Adapter<HttpConnector, Body> {
             HeaderName::from_static("x-amzn-lambda-context"),
             HeaderValue::from_bytes(serde_json::to_string(&lambda_context)?.as_bytes())?,
         );
+
+        if !self.authorization_source.is_empty() && req_headers.contains_key(&self.authorization_source) {
+            let original = req_headers.remove(&self.authorization_source).unwrap();
+            req_headers.insert("authorization", original);
```

Comment:
HeaderMap is case-insensitive: https://docs.rs/http/latest/http/header/index.html
> Header names are normalized to lower case.